test/Interop/Cxx/enum/Inputs/scoped-enums.h

```diff
@@ -0,0 +1,19 @@
+enum class E1 { x = 0, y = 2 };
+
+enum class E2 { x, y, z };
+
+enum class E3 : char { x = 0, y = 2 };
+
+enum class E4 : unsigned int { x = 0, y = 2 };
+
+enum class E5 : unsigned long { x = 0, y = 2 };
+
+enum class E6 : char { x, y, z };
+
+enum class E7 : unsigned int { x, y, z };
+
+enum class E8 : unsigned long { x, y, z };
+
+enum class E10 : int { x = -1, y = 0, z = 2 };
+
+enum class E9 : int { x, y, z };
```

Comment:
Would you mind renaming these enums to names that reveal their intention? For example E6 would become CharScopedEnum or so.

Is there a reason why E10 is above E9? :)

Could you also add a test that verifies that for `enum class Foo { a, b = 42, c }` value of `c` is 43?

Thanks!

Reply:
Will do.